students/yushus/session02/series.py

```diff
@@ -0,0 +1,48 @@
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    return sum_series(n, 0, 1)
+
+def lucas(n):
+    """ compute the nth Lucas number """
+    return sum_series(n, 2, 1)
+
```

Comment:
I like that you took it to this abstraction level and are using your generalized sum_series function to compute fibonaci and lucas, set with the respective initial elements.
Well done!